### PR TITLE
fix: `Intents.all` returning the wrong value

### PR DESCRIPTION
**Pull request by Aityz**
## Summary

This pull request fixes [Issue 2195](https://github.com/Pycord-Development/pycord/issues/2195), with reusable code that gets every flag and removes the unused flags, instead of getting the highest flag.

## Information

- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [x] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Review by Lulalaby — DISMISSED** [review]

File: discord/flags.py [hunk 1]
```diff
@@ -636,8 +636,11 @@ def __init__(self, **kwargs: bool):
     @classmethod
     def all(cls: type[Intents]) -> Intents:
         """A factory method that creates a :class:`Intents` with everything enabled."""
-        bits = max(cls.VALID_FLAGS.values()).bit_length()
-        value = (1 << bits) - 1
+        shifts = list(
+            dict.fromkeys([flag.bit_length() - 1 for flag in cls.VALID_FLAGS.values()])
+        )
```
Comment (Dorukyum) [inline discord/flags.py:641]: I don't understand the point of `list(dict.fromkeys(...))`

Comment (Dorukyum) [reply discord/flags.py:641]: @Aityz could you comment on this?

**Review by Dorukyum — APPROVED** [review]
Tested

**Review by Dorukyum — APPROVED** [review]
The merge commit dismissed my review

**Review by Lulalaby — APPROVED** [review]

**Pull request merged**